### PR TITLE
improve(developer-documentation): Re-organizing and rewriting developer tutorials

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: docs/synthetic-tokens/explainer.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ This smart contract can be deployed using UMA’s synthetic token contract templ
 - token expiration timestamp
```
Comment (nicholaspai) [inline docs/synthetic-tokens/explainer.md:38]: I think the `Priceless Synthetic Tokens` section should explicitly clarify that the rest of the page is talking about one particular flavor of synthetic tokens, the Expiring MultiParty

File: docs/build-walkthrough/build-process.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Development Guide
+sidebar_label: Development Guide
+---
+
+To deploy a synthetic token and start receiving developer mining rewards, it recommended that you follow the docs tutorials in this order.
```
Comment (Tommy1231232) [inline docs/build-walkthrough/build-process.md:6]: minor grammar - "it recommended" to "it is recommended"

File: docs/getting-started/welcome.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ If you would like to learn more about the concepts behind the DVM and priceless
 
 ## For Developers
 
-If you are a developer interested in building with UMA’s technology, start with this [explainer](synthetic-tokens/explainer.md) of how UMA's priceless synthetic tokens work, and then try out this [tutorial](tutorials/mint-locally.md) for how to deploy your own token on testnet. You can find additional reference materials, including a list of UMA-supported mainnet and testnet deployments, [here](dev-ref/addresses.md).
+If you are a developer interested in building with UMA’s technology, start with this [explainer](synthetic-tokens/explainer.md) of how UMA's priceless synthetic tokens work, and then try out this [tutorial](build-walkthrough/mint-locally) for how to deploy your own token on testnet. You can find additional reference materials, including a list of UMA-supported mainnet and testnet deployments, [here](dev-ref/addresses.md).
 
 ## For Market Makers
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:28]: Can we delete this as it isn't one of our primariy users (e.g., token holder, developer)?

File: docs/synthetic-tokens/explainer.md (hunk 1 above)
Comment (Tommy1231232) [reply docs/synthetic-tokens/explainer.md:38]: A few updates to the layout: 

-Within "Overview" section, lets rename "Synthetic tokens" to "Priceless contracts". This lines up better with our website and this way we don't have two sections labeled "Synthetic tokens". 

-Within "Overview" rename "Architecture Overview" to "UMA Overview" (it isn't really an architecture overview)

Comment (Tommy1231232) [reply docs/synthetic-tokens/explainer.md:38]: -Within section "Overviews" rename the top header that says "Overviews" to "Understanding UMA"

File: docs/getting-started/welcome.md (hunk 3 above)
Comment (smb2796) [reply docs/getting-started/welcome.md:28]: Sorry, not sure I understand. Delete the whole section? Or edit the 'build-walkthrough' link?

File: docs/build-walkthrough/build-process.md (hunk 2 above)
Comment (smb2796) [reply docs/build-walkthrough/build-process.md]: Good catch, thanks! Addressed

File: docs/synthetic-tokens/explainer.md (hunk 1 above)
Comment (smb2796) [reply docs/synthetic-tokens/explainer.md:38]: Sections renamed - although I left architecture overview as is.

Also addressed Nick's comments by adding some clarification that the EMP is only one example of a type of synthetic token that could be built on UMA.

**Pull request merged**